docs/src/app/components/PropTypeDescription.jsx

```diff
@@ -51,7 +53,9 @@ const PropTypeDescription = React.createClass({
     }
 
     return (
-      <MarkdownElement text={text} />
+      <div className="propsTable">
```

Comment:
I'm gonna do it, but I think that way too verbose. `propTypeTable` maybe? But one is good as another, I think ;-)